kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.channel = None
         self.exit_stack = contextlib.AsyncExitStack()
+        self.token = ""
```

Comment:
Why do we want to overwrite whatever we got as part of `*args` and was set in `__init__` in BaseVSSClient?